ciphers/rsaBig/RSAcipher(Big).go

```diff
@@ -1,10 +1,10 @@
-package main
+package bigrsacipher
```

Comment:
The test for this file passes but because of the automated command in the golangtest.yml file, this folder is also skipped because it shares the same prefix as rsa folder.